### PR TITLE
Add SmartThings Fan platform

### PR DESCRIPTION
**Pull request by andrewsayre**
**Description:**
Adds documentation for the SmartThings Fan platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20681

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/smartthings.fan.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: "SmartThings Fan"
+description: "Instructions on setting up Samsung SmartThings fans within Home Assistant."
+date: 2018-02-01 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: samsung_smartthings.png
+ha_category: Fan
+ha_release: ""
```
Comment (arsaboo) [inline source/_components/smartthings.fan.markdown]: `0.87`

Comment (klaasnicolaas) [reply source/_components/smartthings.fan.markdown]: Will the parent PR still be included in the 0.87 release?

Comment (arsaboo) [reply source/_components/smartthings.fan.markdown]: That is the hope (and looks like it should be) and we can update it if it doesn't :)

Comment (andrewsayre) [reply source/_components/smartthings.fan.markdown]: I'd prefer to see the `light` prioritized as that will be more valuable to the users and that PR fixes/standardizes a few things this one is assuming.  Getting both in would be good too though. :)

Comment (andrewsayre) [reply source/_components/smartthings.fan.markdown]: Updated it to 0.87, assuming it will go in.

**Review by fabaff — CHANGES_REQUESTED** [review]
See #8389

**Pull request closed without merging**